views/index.pug

```diff
@@ -11,30 +11,38 @@ html
     meta(http-equiv="Content-Type", content="text/html; charset=utf-8")
     meta(name="viewport", content="width=device-width, initial-scale=1")
     title #{__('TITLE', community)}
+    link(href="//static.uqcs.org/bulma/main.css" rel="stylesheet" type="text/css")
     link(href="css/style.css", rel="stylesheet", type="text/css")
-    link(href="//fonts.googleapis.com/css?family=Lato:300,400,700,900,700italic|Open+Sans:700italic,400,600,300,700,800", rel="stylesheet", type="text/css")
   body
-    #wrapper
-      .main
-        .header
-          img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", class="logo")
-          h2 #{__('HEADER', community)}
-        .content
-          .information
+    section(class="section")
+      .container
+        .columns.is-centered.is-mobile
+          .column#body
+            .my-1#header
+              img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", id="logo")
+              h4(class="title is-4 my-5") #{__('HEADER', community)}
             form(method="POST", action="invite")#join-form.form
-              input(type="email", name="email", autofocus, placeholder=__('ENTER_EMAIL'))#slack-email.field
-              if tokenRequired
-                input(type="text", name="token", placeholder=__('ENTER_TOKEN'))#slack-token.field
+              .field
+                label(class="label" for="slack-email") Email Address
+                .control
+                  input(class="input", type="email", name="email", required, autofocus, placeholder="a.smith@example.com")#slack-email.field
+                p(class="help") Valid UQ student email formats are: 
+                  span first.last@uqconnect.edu.au, 
+                  span first.last@uq.net.au, 
+                  span s1234567@student.uq.edu.au. 
               if !!recaptchaSiteKey
-                div(class="g-recaptcha", data-sitekey=recaptchaSiteKey)
-              input(type="submit", value="Join").submit
-        .content
-          .information
-            p Already a slack user? 
+                .field
+                  .control
+                    div(class="g-recaptcha", data-sitekey=recaptchaSiteKey)
```

Comment:
To avoid the inconsistent colour and width of the recaptcha, would it be possible to use the [invisible recaptcha](https://developers.google.com/recaptcha/docs/invisible)? It would work in the background and only present a challenge on clicking.